src/cache/assignment-cache.ts

```diff
@@ -40,16 +40,20 @@ export abstract class AbstractAssignmentCache<T extends Map<string, string>>
     return cachedValue === getMD5Hash(key.variationKey);
   }
 
-  private get(key: AssignmentCacheKey): string | undefined {
+  get(key: AssignmentCacheKey): string | undefined {
     return this.delegate.get(this.toCacheKeyString(key));
   }
 
   set(key: AssignmentCacheKey): void {
     this.delegate.set(this.toCacheKeyString(key), getMD5Hash(key.variationKey));
   }
 
-  private toCacheKeyString({ subjectKey, flagKey, allocationKey }: AssignmentCacheKey): string {
-    return [`subject:${subjectKey}`, `flag:${flagKey}`, `allocation:${allocationKey}`].join(';');
+  entries(): AssignmentCacheKey[] {
```

Comment:
should this be named `keys(),` it seems that is what it is doing--getting the (components) of the cache keys.